### PR TITLE
add parent guid for hybrid dock

### PR DESCRIPTION
**Pull request by chlee75**
- Code fix
 Add Parent GUID for hyrbid dock

### CODE REVIEW

**Comment by superm1** [comment]
Commit message should have a trivial: prefix

**Comment by hughsie** [comment]
> Commit message should have a trivial: prefix

Either `trivial: ` or `ccgx: ` -- to make this kind of change switch to wip/chlee75/ccgx-vidpid and then do `git commit --amend` -- then do `git push --force` to update this PR. Thanks.

**Comment by chlee75** [comment]
Add "ccgx:" prefix to commit message.
Thanks.

File: plugins/ccgx/ccgx.quirk [hunk 1]
```diff
@@ -19,14 +19,18 @@ UpdateMessage = Run firmware update again to upgrade the primary firmware.
...
 GType = FuCcgxHidDevice
+ParentGuid = USB\VID_17EF&PID_1028
 
 [DeviceInstanceId=USB\VID_04B4&PID_5218]
 Plugin = ccgx
 GType = FuCcgxHpiDevice
 
 [DeviceInstanceId=USB\VID_04B4&PID_5218&SID_1F00&APP_6462]
 ImageKind = dual-symmetric
+ParentGuid = USB\VID_17EF&PID_1028
```
Comment (hughsie) [inline plugins/ccgx/ccgx.quirk:35]: Is this one true? Does the `Lenovo Hybrid Dock` have the same USB controller PID as the Gen2 Dock?

Comment (chlee75) [reply plugins/ccgx/ccgx.quirk:35]: Lenovo Hybrid dock has same PD controller comparing to Lenovo Gen2 Dock.
But FW type different.
Gen2 use asymmetric while Hybrid dock use symmetric FW.
In Hybrid Dock, Top most USB is 17ef:1028 Lenovo USB2.0 Hub.

See below USB log of Hybrid dock.

```
[ryan@localhost build]$ lsusb
Bus 002 Device 001: ID 1d6b:0003 Linux Foundation 3.0 root hub
Bus 001 Device 007: ID 138a:0017 Validity Sensors, Inc. VFS 5011 fingerprint sensor
Bus 001 Device 006: ID 8087:0a2b Intel Corp. 
Bus 001 Device 098: ID 05af:1012 Jing-Mold Enterprise Co., Ltd USB Keyboard
Bus 001 Device 010: ID 046d:c534 Logitech, Inc. Unifying Receiver
Bus 001 Device 009: ID 04f2:b52c Chicony Electronics Co., Ltd Integrated Camera
Bus 001 Device 080: ID 04b4:5218 Cypress Semiconductor Corp. USB2.0 Hub             
Bus 001 Device 078: ID 1a40:0101 Terminus Technology Inc. Hub
Bus 001 Device 077: ID 17ef:1029 Lenovo 
Bus 001 Device 076: ID 17ef:1028 Lenovo USB2.0 Hub             
Bus 001 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
[ryan@localhost build]$ lsusb -t
/:  Bus 02.Port 1: Dev 1, Class=root_hub, Driver=xhci_hcd/6p, 5000M
/:  Bus 01.Port 1: Dev 1, Class=root_hub, Driver=xhci_hcd/12p, 480M
    |__ Port 1: Dev 76, If 0, Class=Hub, Driver=hub/4p, 480M
        |__ Port 4: Dev 77, If 0, Class=Hub, Driver=hub/4p, 480M
            |__ Port 2: Dev 78, If 0, Class=Hub, Driver=hub/4p, 480M
                |__ Port 1: Dev 80, If 1, Class=Vendor Specific Class, Driver=, 12M
                |__ Port 1: Dev 80, If 2, Class=Vendor Specific Class, Driver=, 12M
                |__ Port 1: Dev 80, If 0, Class=Vendor Specific Class, Driver=, 12M
    |__ Port 2: Dev 10, If 0, Class=Human Interface Device, Driver=usbhid, 12M
    |__ Port 2: Dev 10, If 1, Class=Human Interface Device, Driver=usbhid, 12M
    |__ Port 6: Dev 98, If 0, Class=Human Interface Device, Driver=usbhid, 1.5M
    |__ Port 6: Dev 98, If 1, Class=Human Interface Device, Driver=usbhid, 1.5M
    |__ Port 7: Dev 6, If 0, Class=Wireless, Driver=btusb, 12M
    |__ Port 7: Dev 6, If 1, Class=Wireless, Driver=btusb, 12M
    |__ Port 9: Dev 7, If 0, Class=Vendor Specific Class, Driver=, 12M
    |__ Port 10: Dev 9, If 0, Class=Video, Driver=uvcvideo, 480M
    |__ Port 10: Dev 9, If 1, Class=Video, Driver=uvcvideo, 480M

```

**Pull request merged**